runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridOverlay.java

```diff
@@ -54,6 +53,10 @@
 	private final Client client;
 	private final ItemManager itemManager;
 
+	private static final int initialMousePointAreaRadius = 5;
+	private static Rectangle initialMousePointArea = null;
+	private static boolean passedInitialMousePointArea = false;
```

Comment:
Why are these static?

Reply:
They are static because they need to retain the value between function calls.

Reply:
but render isnt static.

Reply:
It doesn't need to be `static` to retain value across function calls, it just needs to not be `final`.